docs/clients/rgb_led.rst

```diff
@@ -0,0 +1,132 @@
+
+.. include:: ../notes/c-full-code.rst
+
+.. code-block:: c++
+
+    #include "generic_interface.hpp"
+    #include "rgb_led.hpp"
+
+
+    void main(){
+        // Make a communication interface object
+        GenericInterface com;
+
+        // Make an RGB LED Interface object with obj_id 0
+        RgbLedClient rgbInterface(0);
+
+        // Use the RGB LED Interface Client
+        rgbInterface.red_.get(com)
+
+        // Insert code for interfacing with hardware here  
+    }
+
+Matlab
+~~~~~~
+
+To use the RGB LED Interface client in Matlab, all Vertiq communication code must be included in your
+path. This allows the creation of a RgbLedClient object. See the Message Table below for available messages.
+All message strings use the Short Names. All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the RgbLedClient is:
+
+.. code-block:: Matlab
+
+    % Make a communication interface object
+    com = MessageInterface(’COM18’,115200);
+
+    % Make a RgbLedClient object with obj_id 0
+    RgbInterface = RgbLedClient(’com’,com);
+
+    % Use the RgbLedClient object
+    red = RgbInterface.get(’red’);
+
+Python
+~~~~~~
+
+To use the RGB LED Interface Client in Python, import ``iqmotion`` and create a module that has the RGB LED Interface Client within its firmware. 
+See the table below for available messages. All message strings use the Short Names. 
+All messages use the standard Get/Set/Save functions.
+
+A minimal working example for the RGB LED Interface Client is:
+
+.. code-block:: Python
+    :substitutions:
+    
+    import iqmotion as iq
+
+    com = iq.SerialCommunicator("/dev/ttyUSB0")
+    |variable_name| = iq.|module_name|(com, 0, firmware="speed")
+    
+    red = |variable_name|.get("rgb_led", "red") 
+    print(f"Red Intensity: {red}")
+
+.. _rgb_led_message_table:
+
+Message Table
+~~~~~~~~~~~~~
+
+Type ID 100 | RGB LED Interface
+    .. table:: RGB LED Interface
+        :widths: 8 18 15 10 5 50
+        :class: tight-table
+
+        +--------+---------------+----------------+-----------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | Sub ID | Short Name    | Access         | Data Type | Unit                | Note                                                                                                                                                      |
+        +========+===============+================+===========+=====================+===========================================================================================================================================================+
+        | 0      | red           | get, set, save | uint8     |                     | The intensity of the red LED [0, 255]                                                                                                                     |
```

Comment:
I may specify here and for the other colors and the white intensity that saving this will determine what each of these will be on a reset. May seem a bit obvious, but I find it can be a bit unclear that this will also determine what you start up as for the LEDs, not just what you are instantaneously, and being more clear is better than not with customers. Don't want someone to be surprised when they turn on their white LED and it hits them with 100% intensity because they saved it like that